source/Calamari.Terraform.Tests/CommandsFixture.cs

```diff
@@ -498,27 +498,18 @@ public async Task AzureIntegration()
             var resourceGroupName = AzureTestResourceHelpers.GetResourceGroupName();
             var resourceGroupLocation = RandomAzureRegion.GetRandomRegionWithExclusions();
 
+            var subscriptionId = await ExternalVariables.Get(ExternalVariable.AzureAksSubscriptionId, CancellationToken.None);
+            var tenantId = await ExternalVariables.Get(ExternalVariable.AzureAksSubscriptionTenantId, CancellationToken.None);
+            var clientId = await ExternalVariables.Get(ExternalVariable.AzureAksSubscriptionClientId, CancellationToken.None);
+            var clientPassword = await ExternalVariables.Get(ExternalVariable.AzureAksSubscriptionPassword, CancellationToken.None);
```

Comment:
Do these at the top of the test and stop `PopulateVariables` needing to be async